src/lib/util/printer.ml

```diff
@@ -369,10 +369,12 @@ let print_status_unknown ?(validity_mode=true) loc
     ("I don't know","unknown","fg_cyan") loc
     time steps goal
 
+(* TODO: The timeout answer doesn't exist in the SMT-LIB standard.
+   We should change this function. *)
```

Comment:
I leave this comment here to fix it later. We should always print SMT-LIB format answer with SMT-LIB files. For the legacy output format, I have no strong opinion but having more possible answers could be confusing for the final user.

Reply:
It is useful to inform users that an unknown result is due to a timeout being reached and that re-trying with a higher timeout may succeed. Z3 uses a non-standard "timeout" response in that case. The SMT-LIB way of getting this information is to implement `(get-info :reason-unknown)`, but this is out of scope for this PR. I think both "timeout" and "unknown" are acceptable here.

Reply:
Let's keep `unknown` here and we will fix this in another PR. I will open an issue and put a link here.

Reply:
Issue: #834